src/openrct2/object/WallObject.cpp

```diff
@@ -25,7 +25,8 @@ void WallObject::ReadLegacy(IReadObjectContext* context, IStream* stream)
     _legacyType.wall.height = stream->ReadValue<uint8_t>();
     _legacyType.wall.flags2 = stream->ReadValue<uint8_t>();
     _legacyType.wall.price = stream->ReadValue<uint16_t>();
-    _legacyType.wall.scenery_tab_id = stream->ReadValue<uint8_t>();
+    _legacyType.wall.scenery_tab_id = OBJECT_ENTRY_INDEX_NULL;
+    stream->Seek(1, STREAM_SEEK_CURRENT);
```

Comment:
Does it need the `seek` ? It didn't have one before, like all the other places above (e. g. `FootpathItemObject.cpp`). Theseo other places also updated it to 2

Reply:
Yes it does. We are doing the entry. The others were also slipping another entry.